### DISCUSSION TITLE
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by Smartrove** [prisma/prisma#23773] [Q&A]
### Bug description

<img width="728" alt="prismaError" src="https://github.com/prisma/prisma/assets/108435744/02a49967-6a31-46c1-8622-0b4a8ba81636">

### Expected behavior

this error error is coming up anytime I try starting my server, so i only expected the output log no this error

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @Smartrove 👋 

Thank you for raising this question.

Can you try adding `npx prisma generate` to the start script? For example `"start": "npx prisma generate && nodemon server.ts"`? Do  try it out and let us know if it fixes the issue for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
